docs/integrations/sources/zoho-crm.md

```diff
@@ -0,0 +1,137 @@
+| `bigint`              | `string`     | airbyte_type: big_integer |
+| `event_reminder`      | `string`     |                           |
+| `email`               | `string`     | format: email             |
+| `autonumber`          | `string`     | airbyte_type: big_integer |
+| `jsonarray`           | `array`      |                           |
+| `jsonobject`          | `object`     |                           |
+| `multiselectpicklist` | `array`      |                           |
+| `lookup`              | `object`     |                           |
+| `ownerlookup`         | `object`     |                           |
+| `RRULE`               | `object`     |                           |
+| `ALARM`               | `object`     |                           |
+
+Any other data type not listed in the table above will be treated as `string`.
+
+### Features
+
+| Feature                                   | Supported? \(Yes/No\) |
+|:------------------------------------------|:----------------------|
+| Full Refresh Overwrite Sync               | Yes                   |
+| Full Refresh Append Sync                  | Yes                   |
+| Incremental - Append Sync                 | Yes                   |
+| Incremental - Append + Deduplication Sync | Yes                   |
+| Namespaces                                | No                    |
+
+## List of Supported Environments for Zoho CRM
+
+### Production
+
+| Environment | Base URL                |
+|:------------|:------------------------|
+| US          | https://zohoapis.com    |
+| AU          | https://zohoapis.com.au |
+| EU          | https://zohoapis.eu     |
+| IN          | https://zohoapis.in     |
+| CN          | https://zohoapis.com.cn |
+| JP          | https://zohoapis.jp     |
+
+### Sandbox
+
+| Environment | Endpoint                        |
+|:------------|:--------------------------------|
+| US          | https://sandbox.zohoapis.com    |
+| AU          | https://sandbox.zohoapis.com.au |
+| EU          | https://sandbox.zohoapis.eu     |
+| IN          | https://sandbox.zohoapis.in     |
+| CN          | https://sandbox.zohoapis.com.cn |
+| JP          | https://sandbox.zohoapis.jp     |
+
+### Developer
+
+| Environment | Endpoint                           |
+|:------------|:-----------------------------------|
+| US          | https://developer.zohoapis.com     |
+| AU          | https://developer.zohoapis.com.au  |
+| EU          | https://developer.zohoapis.eu      |
+| IN          | https://developer.zohoapis.in      |
+| CN          | https://developer.zohoapis.com.cn  |
+| JP          | https://developer.zohoapis.jp      |
+
+For more information about available environments, please visit [this page](https://www.zoho.com/crm/developer/sandbox.html?src=dev-hub)
+
+### Performance considerations
+
+If you experience the long time for sync operation, please consider using smaller date range by tuning `start_date` parameter.
```

Comment:
suggest removing this because we should find a way to make the syncs faster. the user shouldn't have to do this ;) 
```suggestion
```

Reply:
ok, done